### DISCUSSION TITLE
Send OTP

### DISCUSSION THREAD

**Discussion by dark-people-zero** [namdevel/ovoid-API#33] [General]
hallo bang, sekarang pas kirim otp. dia ngasih nya link. nah untuk dapet tokennya gimana yah ?

**Comment by namtxs** [comment]
kan memang link, 
otp code nya itu
?code=DINISI

**Comment by namtxs** [comment]
coba cek ini #10

**Reply by dark-people-zero** [reply]
Oh oke gan terima kasih
